upstream/http_json_rpc_client.go

```diff
@@ -798,6 +776,16 @@ func extractJsonRpcError(r *http.Response, nr *common.NormalizedResponse, jr *co
 					details,
 				),
 			)
+		} else if common.EvmDataIsOutOfRangeForFullNodeError(err) {
+			return common.NewErrEndpointDataOutOfRangeForFullNode(
+				common.NewErrJsonRpcExceptionInternal(
+					int(code),
+					common.JsonRpcErrorMissingData,
+					err.Message,
+					nil,
+					details,
+				),
+			)
```

Comment:
Will this be necessary? This error is not coming from upstreams themselves (from client) it happens on higher level, on network.forward